quinn-h3/src/server.rs

```diff
@@ -164,6 +164,15 @@ impl Future for Connecting {
     }
 }
 
+impl From<quinn::Connecting> for Connecting {
+    fn from(connecting: quinn::Connecting) -> Self {
+        Self {
+            connecting,
+            settings: Settings::default(),
+        }
+    }
+}
+
```

Comment:
Should there also be an explicit constructor that allows custom `Settings` to be specified?

Reply:
Made this possible in `Connecting::from_quic()`.